api/src/main/java/pl/kolendateam/dadcard/characterCard/dto/AbilityDTO.java

```diff
@@ -0,0 +1,27 @@
+package pl.kolendateam.dadcard.characterCard.dto;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class AbilityDTO {
```

Comment:
ma sens według Ciebie tworzyć AbilityDTO? I tą HashMape w AbilityMapperDTO?, Wystarczy tą klasę Abilitys co ty zrobiłeś.

Reply:
DTO tak bo jatu wszedzie przypisuje poprostu 5 a nie to co mi w request przyszłó. wystawilem ten pull request by szybko pokazac ci jak pracowac z json w spring przy użyciu hibernate i liqibase. AbilityMapperDTO nie bo to request pobierajacy dane. według sztuki  ten request nie powinien zwracac obiektu. Według CQS - jak cos zapisuje to nie zwraca danych, jak zwraca dane to nie zapisuje. Jedyne co moge/powinienem zwracać to id stworzonego elementu/url do nastepnego requestu (ale nie widziałem by ktoś zwracał url np. w ten restfull sposób jest zrobione https://pokeapi.co/)